main.js

```diff
@@ -1884,27 +2053,57 @@ function getInstances() {
                     continue;
                 }
 
-                logger.debug('host.' + hostname + ' check instance "' + doc.rows[i].id  + '" for host "' + instance.common.host + '"');
-                console.log('host.' + hostname + ' check instance "' + doc.rows[i].id  + '" for host "' + instance.common.host + '"');
+                logger.debug(hostLogPrefix + ' check instance "' + doc.rows[i].id  + '" for host "' + instance.common.host + '"');
+                console.log(hostLogPrefix + ' check instance "' + doc.rows[i].id  + '" for host "' + instance.common.host + '"');
 
-                if (_ipArr.indexOf(instance.common.host) !== -1 || instance.common.host === hostname) {
-                    procs[instance._id] = procs[instance._id] || {};
-                    procs[instance._id].config = JSON.parse(JSON.stringify(instance));
-                    if (instance.common.enabled && (!instance.common.webExtension || !instance.native.webInstance)) count++;
-                }
+                if (checkAndAddInstance(instance, _ipArr) && instance.common.enabled && (!instance.common.webExtension || !instance.native.webInstance)) count++;
             }
 
             if (count > 0) {
-                logger.info('host.' + hostname + ' starting ' + count + ' instance' + (count > 1 ? 's' : ''));
+                logger.info(hostLogPrefix + ' starting ' + count + ' instance' + (count > 1 ? 's' : ''));
             } else {
-                logger.warn('host.' + hostname + ' does not start any instances on this host');
+                logger.warn(hostLogPrefix + ' does not start any instances on this host');
             }
         }
 
         initInstances();
     });
 }
 
+function instanceRelevantForThisHost(instance, _ipArr) {
```

Comment:
Short comment what this function does pls. For which cases does it return true, when false?

Reply:
done